lib/core/seleniumRunner.js

```diff
@@ -9,6 +9,7 @@ const log = require('intel').getLogger('browsertime'),
   BrowserError = require('../support/errors').BrowserError,
   getViewPort = require('../support/getViewPort');
 
+const assert = require('assert');
```

Comment:
This should be used only in test code.  Just a regular `if(...) { throw ... }` should be good.